app/domains/page/profile_page/show_user_with_profile_domain.rb

```diff
@@ -1,12 +1,12 @@
 module Page::ProfilePage
   class ShowUserWithProfileDomain < ApplicationDomain
-    def initialize(user_id:)
-      super()
-      @user_id = user_id
+    def initialize
+      super
     end
 
-    def execute
-      User.includes(:profile).find(@user_id)
+    def execute(user_id: Current.current_user&.id)
+      user = User.includes(:profile, :followers, :following).find(user_id)
```

Comment:
存在しないユーザーidが来たらエラーを吐くようにして欲しいので、
```suggestion
      user = User.includes(:profile, :followers, :following).find!(user_id)
```
にして欲しい